src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -198,15 +198,15 @@ public async Task<ArrayList> WaitForRadioSilenceAsync(TimeSpan? max = null, uint
         public object ReceiveOne(TimeSpan? max = null,CancellationToken cancellationToken = default)
         {
             return ReceiveOneAsync(max, cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.